conf/cassandra.yaml

```diff
@@ -2368,3 +2368,104 @@ drop_compact_storage_enabled: false
 #   compatibility mode would no longer toggle behaviors as when it was running in the UPGRADING mode.
 #
 storage_compatibility_mode: NONE
+
+# Configuration for AutoRepair Scheduler
+#auto_repair:
+#  # Enable/Disable auto repair scheduler.
+#  # If it is set to false, then the scheduler thread will not be spun up.
+#  # If it is set to true, then the repair scheduler thread will be created. The thread will
+#  # look for secondary config available for each repair type (full, incremental, and preview_repaired),
+#  # and based on that, it will schedule repairs.
+#  enabled: false
+#  # Time interval between successive checks for repair scheduler to check if either the ongoing repair is completed or if
+#  # none is going, then check if it's time to schedule or wait.
+#  repair_check_interval: 5m
+#  # Maximum number of retries for a repair session.
+#  repair_max_retries: 3
+#  # Backoff time in seconds for retrying a repair session.
```

Comment:
nit, Don't need to say "in seconds" when it accepts a duration

Reply:
Done